src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ShowPlan/Node.cs

```diff
@@ -595,7 +590,8 @@ public string[] GetDisplayLinesOfText(bool roundCostForSmallGraph = false)
 
             if (!this.HasPDWCost || cost > 0)
             {
-                if(roundCostForSmallGraph && this.graph != null && this.graph.NodeStmtMap.Count < 20){
+                if (roundCostForSmallGraph && this.graph != null && this.graph.NodeStmtMap.Count < 20)
```

Comment:
Why 20? It'd be good to make this into a constant and then add a comment explaining where that value comes from so it's clear what it is and how it's being used.

Reply:
For larger plans, we wanted to show the cost in decimals. For that, we have decided the threshold as 20 for now.

Reply:
pls add comment in the code about why 20 is picked.

Reply:
And use a const for magic values like this - it makes it easier to understand if you have a variable with a clear name